### DISCUSSION TITLE
"About" dialog could show Cockpit and its version.

### DISCUSSION THREAD

**Discussion by mvollmer** [rhinstaller/anaconda-webui#201] [General]
The "About" dialog shows that the installer uses Anaconda, and the version of Anaconda. I think it should also contain Cockpit and its version.

**Comment by KKoukiou** [comment]
https://issues.redhat.com/browse/INSTALLER-4276

We don't use discussions for RFE. Please use bugzilla or Jira. We do have some  Jira ticket for the Anaconda part.

**Reply by mvollmer** [reply]
I have added a comment to https://issues.redhat.com/browse/INSTALLER-4276.
